### PR TITLE
Translation update for the da_DK locale

### PR DESCRIPTION
**Pull request by tgc**
A full Danish translation of OMP made by @neffe.

To complete the translation of OMP 3.1.2 these two PRs are also needed:
https://github.com/pkp/pkp-lib/pull/5077
https://github.com/pkp/usageStats/pull/17

### CODE REVIEW

File: plugins/importexport/csv/locale/da_DK/locale.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  * locale/da_DK/locale.xml
+  *
+  * Copyright (c) 2014-2019 Simon Fraser University
+  * Copyright (c) 2003-2019 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings.
+  -->
+
+<locale name="da_DK" full_name="Dansk">
```
Comment (asmecher) [inline plugins/importexport/csv/locale/da_DK/locale.xml]: Some English text crept into this one.

**Comment by asmecher** [comment]
Thanks, @tgc! One file looks like it didn't get translated, but otherwise it looks good. That's a lot of work!

**Comment by tgc** [comment]
The missing translation should now be taken care of thanks to @neffe.
I squashed in the changes along with a couple of missing translations in the email templates.

File: plugins/importexport/csv/locale/da_DK/locale.xml (hunk 1 above)
Comment (tgc) [reply plugins/importexport/csv/locale/da_DK/locale.xml]: Should now be gone thanks to @neffe.

**Comment by asmecher** [comment]
Marvelous, thanks, @tgc and @neffe! Merged and cherry-picked forward to `master`.

**Pull request merged**